modules/misc/nix.nix

```diff
@@ -10,6 +10,20 @@ let
 
   isNixAtLeast = versionAtLeast (getVersion nixPackage);
 
+  nixPath = lib.concatStringsSep ":" cfg.nixPath;
+
+  # The deploy path for declarative channels. The directory name is prefixed
+  # with a number to make it easier for files in ~/.nix-defexpr to control the
+  # order they'll be read relative to each other.
+  channelPath = ".nix-defexpr/50-home-manager";
```

Comment:
What about the path one should use with an XDG-compliant `nix` (like e.g: #3876)?